backend/kernelCI_app/views/treeTestsView.py

```diff
@@ -39,13 +39,18 @@ def extract_error_message(self, misc: Union[str, dict, None]):
         return "unknown error"
 
     def get(self, request, commit_hash: str | None):
+        origin_param = request.GET.get("origin")
+        git_url_param = request.GET.get("git_url")
+        git_branch_param = request.GET.get("git_branch")
         path_param = request.GET.get('path')
         if path_param:
+            # TODO: 'boot' and 'boot.', right now is only using 'boot.'
             path_filter = "AND t.path LIKE %s"
-            params = [commit_hash, f'{path_param}%']
+            params = [commit_hash, origin_param, git_url_param, git_branch_param,
```

Comment:
we can create this guy `params = [commit_hash, origin_param, git_url_param, git_branch_param]` before the `if` and just append  `path_param` here